### DISCUSSION TITLE
translation/ebpf-solve-service-mesh-sidecar/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#32] [Announcements]
# translation/ebpf-solve-service-mesh-sidecar/

本文回顾了Linux 内核的连接性，实现服务网格的几种模式，以及如何使用 eBPF 实现无 Sidecar 的服务网格。

https://lib.jimmysong.io/translation/ebpf-solve-service-mesh-sidecar/

**Comment by rootsongjc** [comment]
现在说告别 Sidecar 为时过早，请大家看下[请暂时抛弃使用 eBPF 取代服务网格和 sidecar 模式的幻想](https://jimmysong.io/blog/epbf-sidecar-and-service-mesh/)这篇文章再做决定。
